Cargo.toml

```diff
@@ -29,7 +29,22 @@ unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 [dev-dependencies]
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
-libipld = { version = "0.15.0", default-features = false, features = ["dag-cbor"] }
-libp2p = { version = "0.50.0", features = ["tcp", "noise", "yamux", "rsa", "async-std"] }
-multihash = { version = "0.17.0", default-features = false, features = ["blake3", "sha2"] }
-tracing-subscriber = { version = "0.3.5", features = ["env-filter", "tracing-log"] }
+libipld = { version = "0.15.0", default-features = false, features = [
+    "dag-cbor",
+] }
+libp2p = { version = "0.51.2", features = [
+    "tcp",
+    "noise",
+    "yamux",
+    "rsa",
+    "ed25519",
+    "async-std",
+] }
+multihash = { version = "0.17.0", default-features = false, features = [
```

Comment:
@mxinden I’m slightly confused: the other PR #46 upgraded to 0.18 and only one of those choices can be correct.

Reply:
`rust-libp2p` `v0.51.3` uses `multihash` `v0.17.0`. 

https://github.com/libp2p/rust-libp2p/blob/3c5940aeadb9ed8527b6f7aa158797359085293d/Cargo.lock#L3259-L3273